pinot-core/src/main/java/org/apache/pinot/core/query/request/context/QueryContext.java

```diff
@@ -350,6 +381,7 @@ public String toString() {
     private List<ExpressionContext> _selectExpressions;
     private List<String> _aliasList;
     private FilterContext _filter;
+    private ExpressionContext _filterExpression;
```

Comment:
The change in the `Builder` is not necessary

Reply:
Fixed